zabbix_auto_config/processing.py

```diff
@@ -666,11 +677,27 @@ def do_update(self):
             # if api ver >= 4.4
             if "inventory_mode" in zabbix_host.keys():
                 if zabbix_host["inventory_mode"] != "1":
-                    self.set_inventory(zabbix_host, 1)
+                    self.set_inventory_mode(zabbix_host, 1)
             # else api ver. < 4.4
             else:
                 if zabbix_host["inventory"]["inventory_mode"] != "1":
-                    self.set_inventory(zabbix_host, 1)
+                    self.set_inventory_mode(zabbix_host, 1)
+
+            if db_host["inventory"]:
+                if zabbix_host["inventory"]:
+                    inventories_to_add = dict(set(db_host["inventory"].items()).difference(set(zabbix_host["inventory"].items())))
+                else:
+                    inventories_to_add = db_host["inventory"]
+
+                if inventories_to_add:
+                    # inventory outside of zac management
+                    ignored_inventories = {k: v for k, v in inventories_to_add.items() if k not in self.managed_inventories}
+                    # inventories managed by zac and to be updated
+                    inventories = {k: v for k, v in inventories_to_add.items() if k in self.managed_inventories}
+                    if inventories:
+                        self.set_inventories(zabbix_host, inventories)
+                    else:
+                        logging.debug("Zac is not configured to manage inventory properties: '%s'.", ignored_inventories)
```

Comment:
I don't think this should be a if else? They're not mutually exclusive? You could have a inventory to set _and_ "unmanaged" properties.  You will loose a log message in that case.

Also: I think that should be considered a warning level event. It's an indicator that something is wrong in a source/configured wrong.

Also: Is really just a single inventory and not "inventories"